src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { User } from '../../api/users';
+
+interface Props {
+  user?: User;
```

Comment:
```suggestion
  user: User;
```
`user` should be required, it doesn't make sense to render `UserInfo` component without user